client/lib/plugins/notices.jsx

```diff
@@ -43,6 +43,20 @@ function getTranslateArg( logs, sampleLog, typeFilter ) {
 }
 
 module.exports = {
+
+	shouldComponentUpdateNotices: function( currentNotices, nextNotices ) {
+		if ( currentNotices.errors && currentNotices.errors.length !== nextNotices.errors.length ) {
```

Comment:
I think we should use immutable objects, so we could compare `currentNotices.errors` to `nextNotices.errors`.